Android/app/src/main/java/com/example/androidgui/inventoryactivities/AddInventoryActivity.java

```diff
@@ -6,27 +6,32 @@
 import android.widget.EditText;
 import androidx.appcompat.app.AppCompatActivity;
 import com.example.androidgui.R;
+import presenter.inventorysystem.AddinventoryViewInterface;
 import presenter.inventorysystem.InventoryManager;
+import presenter.inventorysystem.InventoryPresenter;
 
-public class AddInventoryActivity extends AppCompatActivity implements View.OnClickListener{
+
+public class AddInventoryActivity extends AppCompatActivity implements View.OnClickListener, AddinventoryViewInterface {
     Button button;
     EditText name;
     EditText price;
     EditText amount;
     EditText date;
     EditText freshness;
     final InventoryManager im = new InventoryManager();
```

Comment:
is this also a presenter or is it still a controller? MVP doesn't have controllers, so the methods originally in controller classes should be done in presenter instead